languages/rust/exercises/concept/README.md

```diff
@@ -9,6 +9,7 @@ These are the concept exercises that have currently been implemented, as well as
 | exercise                                                            | concepts                                                            | prerequisites                                                       |
 | ------------------------------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------- |
 | [`enums`][concept-exercise-enums]                                   | `basic-enums`<br/>`conditionals`                            | `strings`                                                     |
+| [`numbers`][concept-exercise-numbers]                                   | `basic-numbers`<br/>`conditionals`                            | `strings`                                                     |
```

Comment:
NOTE: We maintain the concept exercise prerequisites and concepts in a couple places:
1. reference doc
2. the concept exercise issues
3. here

The DRY part of me says we should derive them from the reference doc but I'm not sure.